crates/ruff/src/rules/flake8_annotations/fixes.rs

```diff
@@ -30,6 +30,7 @@ pub(crate) fn add_return_annotation<T: Ranged>(
         if matches!(tok, Tok::Lpar) {
             if count == 0 {
                 seen_lpar = true;
+                seen_rpar = false;
```

Comment:
Can we simplify this even further by using the end of the `Arguments` as the starting range? Then it would just be: find the first colon. We wouldn't even need to lex, I don't think.